talks/_posts/2020-6-16-flannery-cpdd-presentation.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: talk
+title: "Peer influences on adolescents’ intentions for future electronic nicotine delivery system use"
+nickname: 2020-6-16-flannery-cpdd-presentation
+authors: "Flannery JS"
+year: "2020"
+conference: "College on Problems of Drug Dependence"
+image: /assets/images/talks/Flannery_cpdd_2020.png
+projects: []
```

Comment:
it's part of ace, right?

Reply:
yep

Reply:
you can change `projects: []` to `projects: [ace]` then

Reply:
```suggestion
projects: [ace]
```